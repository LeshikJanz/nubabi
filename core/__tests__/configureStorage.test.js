import configureStorage from '../configureStorage';

describe('configureStorage', () => {
  it('setups redux-persist', () => {
    expect(configureStorage('testapp', ['store_path_to_serialize'])).toMatchSnapshot();
  });
});
