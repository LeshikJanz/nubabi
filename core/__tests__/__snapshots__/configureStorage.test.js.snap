// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configureStorage setups redux-persist 1`] = `
Object {
  "debounce": 100,
  "keyPrefix": "testapp:",
  "transforms": Array [
    Object {
      "in": [Function],
      "out": [Function],
    },
    Object {
      "in": [Function],
      "out": [Function],
    },
    Object {
      "in": [Function],
      "out": [Function],
    },
    Object {
      "in": [Function],
      "out": [Function],
    },
  ],
  "whitelist": Array [
    "auth",
    "apollo",
    "babies",
    "growth",
    "settings",
    "viewer",
  ],
}
`;
